stable-diffusion/.projectignore

```diff
@@ -0,0 +1,25 @@
+# MOST projects DO NOT NEED this file. Only those which publish a data/ folder
```

Comment:
Most projects don't need to submit their `.projectignore` file, I think this will apply to this project too.

Reply:
Removed!